Issuetracker : #i56949#
CWS          : N/A
Author       : <maho@openoffice.org> (JCA)
Description  : FreeBSD porting : Add conditional the case of @GCCVER@ >= 30401.
Like GNU/Linux's case, we also need the case for $CVER if @GCCVER@ is equal to or larger than 3.4.1

--- config_office/set_soenv.in	Thu Sep 22 23:30:56 2005
+++ config_office/set_soenv.in	Sat Oct  1 12:59:30 2005
@@ -347,6 +347,9 @@
       $JRELIBDIR      = '$JAVA_HOME'.$ds."jre".$ds."lib".$ds."i386";
       $JRETOOLKITDIR  = '$JAVA_HOME'.$ds."jre".$ds."lib".$ds."i386".$ds."client";
       $JRETHREADDIR   = '$JAVA_HOME'.$ds."jre".$ds."lib".$ds."i386".$ds."native_threads";
+      if( @GCCVER@ >= 30401 ) {
+      	 $CVER        = "C341";
+      }
 } 
 elsif ( $platform =~ m/linux-gnu/ ) 
 {
